app/scenes/login/login.js

```diff
@@ -39,11 +58,57 @@ class Login extends Component {
         }
     }
 
-    signIn() {
+    preSignIn = () => {
+        this.setState({error: null});
+        if (!this.props.loginId) {
+            // it's slightly weird to be constructing the message ID, but it's a bit nicer than triply nested if statements
+            let msgId = 'login.no';
+            if (this.props.config.EnableSignInWithEmail === 'true') {
+                msgId += 'Email';
+            }
+            if (this.props.config.EnableSignInWithUsername === 'true') {
+                msgId += 'Username';
+            }
+            if (this.props.license.IsLicensed === 'true' && this.props.config.EnableLdap === 'true') {
+                msgId += 'LdapUsername';
+            }
+
+            this.setState({
+                error: this.props.intl.formatMessage({id: msgId}, {
```

Comment:
actually `ErrorText` has this

```javascript
return (
            <Text style={GlobalStyles.errorLabel}>
                {this.props.error.message || this.props.error}
            </Text>
        );
```

So I don't think passing a `FormattedText` will work unless we change this